tools/test.py

```diff
@@ -243,6 +245,10 @@ def main():
             cfg.device,
             device_ids=[int(os.environ['LOCAL_RANK'])],
             broadcast_buffers=False)
+
+        if cfg.device == 'npu' and args.tmpdir is None:
+            args.tmpdir = './npu_tmpdir'
```

Comment:
why do we need to specify tmpdir for NPU?

Reply:
in fn collect_results_cpu, if if tmpdir is None, some tesnors will init on cuda by default, and no device choice supported..
I init a tmpdir to avoid error here

Reply:
need comment

Reply:
added